test/sharness/t0130-multinode.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+#
+# Copyright (c) 2015 Jeromy Johnson
+# MIT Licensed; see the LICENSE file in this repository.
+#
+
+test_description="Test multiple ipfs nodes"
+
+. lib/test-lib.sh
+
+IPTB_ROOT=`pwd`/testbed
+
+test_expect_success "set up a few nodes" '
+	iptb -n=3 init &&
```

Comment:
where does iptb put these? it has to be within the current (trashdir) directory. maybe inside `.iptb/`.

(related: https://github.com/whyrusleeping/iptb/issues/2)